api/app/fire_behaviour/cffdrs.py

```diff
@@ -545,12 +545,25 @@ def fine_fuel_moisture_code(ffmc: float, temperature: float, relative_humidity:
 
     if ffmc is None:
         ffmc = NULL
+    if temperature is None:
+        temperature = NULL
+    if relative_humidity is None:
+        relative_humidity = NULL
+    if precipitation is None:
+        precipitation = NULL
+    if wind_speed is None:
+        logger.error("Failed to calculate ffmc")
+        return None
```

Comment:
```suggestion
        wind_speed = NULL
```

Reply:
The R ffmc function will fail if passed a NULL windspeed, so we have to short circuit and return None here. I've added a code comment to make this clear.